Homework/Hang_Nguyen/lesson_1/range-sum-query-immutable.py

```diff
@@ -0,0 +1,20 @@
+class NumArray:
+
+    def __init__(self, nums):
+        self.sumLst = []
+        s = 0
+        for num in nums:
+            s += num
+            self.sumLst.append(s)
+
+    
+    def sumRange(self, left, right):
+        if left > 0 and right > 0:
```

Comment:
do we need `right > 0` ?

Reply:
Hi thay Quang,

I have removed redundant condition. You can help to take a look.

Many thanks,
Hang Nguyen